area4/dividers.txt

```diff
@@ -33,7 +33,7 @@ r(`,,,`)r r(`,,,`)r r(`,,,`)r
 <><><><><><>
 ^,^,^,^,^,^,
 &*&*&*&*&*&*
-Random number divider
```

Comment:
This is ignored, please revert

Reply:
You reverted. Sorry. I just wrote a program to reduce each line to 12 chars.